packages/perseus/src/widgets/interactive-graphs/movable-point.tsx

```diff
@@ -0,0 +1,52 @@
+import Color from "@khanacademy/wonder-blocks-color";
+import {vec, useMovable, useTransformContext} from "mafs";
+import * as React from "react";
+import {useRef} from "react";
+
+type Props = {
+    point: vec.Vector2;
+    onMove: (newPoint: vec.Vector2) => unknown;
+};
+
+export const MovablePoint = (props: Props) => {
+    const hitboxRef = useRef<SVGCircleElement>(null);
+    const {point, onMove} = props;
+
+    const {dragging} = useMovable({
+        gestureTarget: hitboxRef,
+        point,
+        onMove,
+        constrain: (p) => p,
+    });
+
+    const {viewTransform, userTransform} = useTransformContext();
+    const transformToPx = vec.matrixMult(viewTransform, userTransform);
+    const [x, y] = vec.transform(point, transformToPx);
+
+    return (
+        <g
+            ref={hitboxRef}
+            className="movable-point"
+            tabIndex={0}
+            style={
+                {
+                    cursor: dragging ? "grabbing" : "grab",
+                    touchAction: "none",
+                    outline: "none",
+                    "--movable-point-color": Color.blue,
+                } as any
+            }
+        >
+            <circle className="movable-point-hitbox" r={30} cx={x} cy={y} />
+            <circle className="movable-point-halo" r={11} cx={x} cy={y} />
+            <circle className="movable-point-ring" r={8} cx={x} cy={y} />
+            <circle
+                className="movable-point-center"
+                r={6}
```

Comment:
Would it make sense to break out these scalar numbers into a `constants` file? I'm not entirely sure since it doesn't look like there are a lot of reused ones (between this file and the CSS file), but I think naming them could make it easier to figure out why the sizes are what they are in the future.